Example/ViewController.swift

```diff
@@ -504,3 +507,39 @@ extension ViewController: VisualInstructionDelegate {
         return presented
     }
 }
+
+
+extension ViewController: InstructionsCardCollectionDelegate {
+    
+    public func instructionsCardCollection(_ instructionsCardCollection: InstructionsCardCollection, previewFor step: RouteStep) {
```

Comment:
Lets talk sometime about using a standard delegate method to hook up this functionality both in the default example and in this UI VC.

Reply:
sure.